static/css/devhub/base.less

```diff
@@ -1,32 +1,45 @@
 @import '../common/normalize.css';
 @import '../impala/fonts/fira/fira.less';
 
-// Colors
-@primary-font: 'Fira Sans', sans-serif;
-@primary-font-color: #fff;
-@secondary-font-color: #333;
-@tertiary-font-color: #40d38d;
-
-@button-border: #979797;
-@primary-button-background: #29c978;
-
-@primary-section-background: #0568ba;
-@secondary-section-background: #f3f5f8;
-@tertiary-section-background: #1d3770;
-
+// Grid
+@grid-max: 940px;
 @phone-large: ~"only screen and (min-width: 428px)";
 @phone: ~"only screen and (max-width: 768px)";
```

Comment:
Another rationale for using small, medium etc is to bear in mind the change when flipping between landscape + portrait - that might change a device class which starts to make less sense when using specific naming. Whereas a more simple sizing will still make sense.

The overlaps in real terms of viewport by device is pretty crazy. There's a great picture demonstrating that somewhere - I'll see if I can find it.

In most cases you'll probably only need to do some tweaks to larger viewport sizes since the default mobile-first styling should cover mobile and tablets.